src/current/v23.1/security-reference/security-overview.md

```diff
@@ -165,11 +165,11 @@ CockroachDB {{ site.data.products.core }} here refers to the situation of a user
    <td>Network-level Configuration of allowed IP addresses</td>
  </tr>
  <tr>
-  <td>&nbsp;</td>
+  <td>✓<a href="#privatelink-multiregion-serverless"><sup>1</sup></a></td>
```

Comment:
This and the below were misses when PrivateLink for Serverless launched. This table is a little different from the Cloud one.